src/openforms/registrations/contrib/zgw_apis/tests/test_backend.py

```diff
@@ -984,19 +984,26 @@ def test_reference_can_be_extracted(self, m):
 
         self.assertEqual("abcd1234", reference)
 
-    def test_submission_with_zgw_backend_override_attachment_iotype(self, m):
+    def test_submission_with_zgw_backend_override_fields(self, m):
+        """Assert that override of default values for the ZGW backend works"""
         submission = SubmissionFactory.from_components(
             [
                 {
                     "key": "field1",
                     "registration": {
                         "informatieobjecttype": "https://catalogi.nl/api/v1/informatieobjecttypen/10",
+                        "bronorganisatie": "100000009",
+                        "vertrouwelijkheidaanduiding": "zeer geheim",
```

Comment:
```suggestion
                        "vertrouwelijkheidaanduiding": "zeer_geheim",
```
please check against the enum on https://redocly.github.io/redoc/?url=https://raw.githubusercontent.com/VNG-Realisatie/gemma-documentregistratiecomponent/1.2.0/src/openapi.yaml#tag/enkelvoudiginformatieobjecten/operation/enkelvoudiginformatieobject_create

Use the available constants to prevent these sort of mistakes please.